cmd/rosetta/cli.go

```diff
@@ -156,10 +156,10 @@ VERSION:
 		Required: true,
 	}
 
-	cliFlagCustomCurrenciesSymbols = cli.StringSliceFlag{
-		Name:  "custom-currencies",
-		Usage: "Specifies the symbols of enabled custom currencies (i.e. ESDT identifiers).",
-		Value: &cli.StringSlice{},
+	cliFlagConfigFileCustomCurrencies = cli.StringFlag{
+		Name:     "config-custom-currencies",
```

Comment:
We receive the list of custom currencies (and their minimal metadata) by means of a JSON configuration file.